### PR TITLE
Feature namespace refactor post merge fixes

### PR DESCRIPTION
**Pull request by DDReaper**
Overview
---
Minor updates to the namespace feature branch, requested from reviewers

Changes
---
[x] reverting all ProjectSettings assets - using Dev_Working_Branch versions
[x] Verified all images for all .MD files are in the repo
[x] Checked all image links use local and not remote links
[x] Reverted a few /master/ renames that were in error

### CODE REVIEW

File: Assets/MixedRealityToolkit/SpatialUnderstanding/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ or in your Visual Studio Package.appxmanifest capabilities.
 
 ### Microsoft HoloLens Documentation
 - [Spatial Mapping In Unity](https://developer.microsoft.com/en-us/windows/mixed-reality/spatial_mapping_in_unity)
-- [Spatial Understanding In Unity](https://developer.microsoft.com/en-us/windows/mixed-reality/spatial_mapping_in_unity#MixedRealityToolkit.spatialunderstanding)
+- [Spatial Understanding In Unity](https://developer.microsoft.com/en-us/windows/mixed-reality/spatial_mapping_in_unity#holotoolkit.spatialunderstanding)
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/SpatialUnderstanding/README.md]: I think this has actually been updated to https://developer.microsoft.com/en-us/windows/mixed-reality/spatial_mapping_in_unity#higher-level_mesh_analysis:_spatialunderstanding

File: FallCreatorsUpdate.md [hunk 2]
```diff
@@ -43,5 +43,5 @@ If you're looking for **Boundary** tools:
     * MixedRealityCameraParent.prefab
     
 If you're looking for **Motion controller Grab Mechanics**:
-* See  [**MotionControllersGrabMechanics**](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/master/Assets/MixedRealityToolkit-Examples/MotionControllers-GrabMechanics) page.
+* See  [**MotionControllersGrabMechanics**](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/Dev_Working_Branch/Assets/MixedRealityToolkit-Examples/MotionControllers-GrabMechanics) page.
```
Comment (keveleigh) [inline FallCreatorsUpdate.md]: Do we want all these to link to the development branch vs the (likely) more stable master branch?

File: Assets/MixedRealityToolkit/Sharing/README.md [hunk 3]
```diff
@@ -166,7 +166,7 @@ Draws the default Sharing Stage Inspector and adds the SyncRoot Hierarchy view s
 
 #### [SDK](Scripts/SDK)
 ---
-Contains scripts compiled from the native [MixedRealityToolkit\Sharing](https://github.com/Microsoft/MixedRealityToolkit/tree/Dev_Working_Branch/Sharing) repository and using the SWIG tool to generate different language bindings.
+Contains scripts compiled from the native [MixedRealityToolkit\Sharing](https://github.com/Microsoft/MixedRealityToolkit/tree/master/Sharing) repository and using the SWIG tool to generate different language bindings.
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit/Sharing/README.md]: Could we shorten this to just `/Sharing`? like we did above?

File: FallCreatorsUpdate.md (hunk 2 above)
Comment (StephenHodgson) [reply FallCreatorsUpdate.md]: Yeah I think we can just use `/Assets/MixedRealityToolkit-Examples/MotionControllers-GrabMechanics` as the link, correct?

Comment (keveleigh) [reply FallCreatorsUpdate.md]: Oh even better, yeah! Just use the relative links, so they work in either branch

File: Assets/MixedRealityToolkit/Sharing/README.md (hunk 3 above)
Comment (keveleigh) [reply Assets/MixedRealityToolkit/Sharing/README.md]: This one links to a different repo, the non-Unity one

Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/Sharing/README.md]: Oh I see. I wasn't aware we had a Dev_Working_Branch there.

Comment (DDReaper) [reply Assets/MixedRealityToolkit/Sharing/README.md]: There isn't a dev_working_branch on the base MRTK, hence why I had to revert :D

File: FallCreatorsUpdate.md (hunk 2 above)
Comment (DDReaper) [reply FallCreatorsUpdate.md]: I tried, you can have relative links to images. With hyperlinks it seems you can't with GitHub.
As to where they should point, they simply won't work against master until merged there.

In my view, stuff should "Work" in Dev, then when it's merged with master, they should be updated to work with master.   Else you aren't really testing it.

File: Assets/MixedRealityToolkit/SpatialUnderstanding/README.md (hunk 1 above)
Comment (DDReaper) [reply Assets/MixedRealityToolkit/SpatialUnderstanding/README.md]: Updated to new link

File: FallCreatorsUpdate.md (hunk 2 above)
Comment (DDReaper) [reply FallCreatorsUpdate.md]: Ok, tested again and relative hyperlinks do actually work, so I'll update the rest.

Will make them a lot easier to maintain.

Comment (DDReaper) [reply FallCreatorsUpdate.md]: Sorted, hard coded links be gone.
Only Dev_Working_Branch links are those that actually point to the Dev_Working_Branch :D (e.g. to tell you it's the dev place :D)
